spec/defines/nodejs_npm_spec.rb

```diff
@@ -23,348 +21,317 @@
 
     # su user npm install <package>
     context 'with package set to express, user set to foo and target set to /home/npm/packages' do
-      let (:params) {{
-        :package => 'express',
-        :target  => '/home/npm/packages',
-        :user    => 'foo',
-      }}
+      let :params do
+        {
+          :package => 'express',
+          :target  => '/home/npm/packages',
+          :user    => 'foo',
+        }
+      end
 
       it 'the npm install command should run under user foo' do
-        is_expected.to contain_exec('npm_install_express').with({
-          'command' => '/usr/bin/npm install express ',
-          'user'    => 'foo',
-         })
+        is_expected.to contain_exec('npm_install_express').with('command' => '/usr/bin/npm install express ',
+                                                                'user'    => 'foo',)
```

Comment:
If you're interested in going back to the previous format, here's the diff required. You do not need to Modulesync as those block checks actually aren't related (but still good IMO):
````
$ git diff
diff --git a/spec/defines/nodejs_npm_spec.rb b/spec/defines/nodejs_npm_spec.rb
index 88a93d2..b4e27f2 100644
--- a/spec/defines/nodejs_npm_spec.rb
+++ b/spec/defines/nodejs_npm_spec.rb
@@ -30,8 +30,10 @@ describe 'nodejs::npm', :type => :define do
       end

       it 'the npm install command should run under user foo' do
-        is_expected.to contain_exec('npm_install_express').with('command' => '/usr/bin/npm install express ',
-                                                                'user'    => 'foo',)
+        is_expected.to contain_exec('npm_install_express').with(
+          'command' => '/usr/bin/npm install express ',
+          'user'    => 'foo',
+         )
       end
     end
````